CONTRIBUTE.md

```diff
@@ -0,0 +1,104 @@
+
+ *Issue [#7](https://github.com/ffoodd/a11y.css/issues/7) opened by [@7studio](https://twitter.com/7studio) suggested a decent work-around to display messages for those tags, as long as they are followed by a non-self-closing element. Still better than nothing.*
+
+2. Messages are generated through a fixed pseudo-element. However there could be a containing block issue if the parent is a transformed element (`transform`):
+ * [W3C Specification about this](http://www.w3.org/TR/css3-transforms/#transform-property)
+ * [What no one told you about z-index](http://philipwalton.com/articles/what-no-one-told-you-about-z-index/) ([translated in French by @iamvdo](http://blog.iamvdo.me/post/41094013194/comprendre-z-index-et-les-contextes-dempilement))
+
+3. Along the same lines, tests on elements that are contained within `<head>` cannot be displayed (since `<head>` is a hidden element). I need to find a way to do this.
+
+ *Issue [#66](https://github.com/ffoodd/a11y.css/issues/66) opened by [@7studio](https://twitter.com/7studio) (again) helped with this point, thanks to [an idea](https://mathiasbynens.be/notes/css-hidden-elements) from [Mathias Bynens](https://twitter.com/mathias).
+
+4. To avoid cases when outline could be hidden, the property [outline-offset](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset) is being used in order to display it inside the element (rather than outside). Thanks to [@7studio](https://twitter.com/7studio) in [#4](https://github.com/ffoodd/a11y.css/issues/4).
+
+## Automatisation
+
+We all have to stay up-to-date, so you'll find a [Gulp](http://gulpjs.com/) configuration file.
+Some useful tasks are ready-to-run.
+
+If you run `gulp` — which runs `gulp build` behind the scene — you'll compile Sass files (including *sourcemaps*)
+and optimize the CSS output.
+
+You can also launch `gulp docs`, which will generate the whole documentation *via* both SassDoc and Hologram.
+
+Each task can be run individually:
+* `gulp sass`
+* `gulp minify`
+* `gulp sassdoc`
+* `gulp hologram`
+
+And of course you can watch for changes to automatically compile Sass and optimize the CSS output using `gulp watch`.
+
+## Documentation
+
+### Building SassDoc
+
+With global SassDoc install:
+
+```sh
+$ npm install sassdoc -g
+$ sassdoc sass/ --dest docs/ --verbose --config .sassdocrc
+```
+
+With local SassDoc install:
+
+```sh
+$ npm install
+$ node_modules/sassdoc/bin/sassdoc sass/ --dest docs/ --verbose --config .sassdocrc
+```
+
+### Build Hologram
+
+This one is easy. Usign command line, just `cd` to `a11y.css` then run `hologram`.
+Et voilà !
+
+To learn more about Hologram, take a look at [their GitHub repository](https://github.com/trulia/hologram).
+
+
+## Technical structure
```

Comment:
You're right. I'll also add a part talking about csslint and scsslint when I'll update the gulpfile. Contributing will need to run tests and build dicumentations before opening a PR.